addon/authenticators/oidc.js

```diff
@@ -187,7 +188,9 @@ export default BaseAuthenticator.extend({
     refresh_token,
     expires_in,
     id_token
-  }) {
+  },
+  disableTrigger=false
```

Comment:
Please document what this parameter does..

Reply:
Done. However I'd even more like to remove it again. But to my understanding, ember-simple-auth will trigger that event after `authenticate()`, so we probably shouldn't.. Don't wanna mess with it too much if not needed